### ISSUE TITLE
Remove current string

### ISSUE THREAD

**Issue by lucabecchetti** [gilbitron/laravel-vue-pagination#139]
How can i remove (current) string?

**Comment by gilbitron** [comment]
The "current" string should be hidden by bootstraps `.sr-only` class, as it's only for screen readers. To implement this manually you could do:

```css
.sr-only {
    display: none;
}
```
